docs/guidance/EnterpriseScaleAnalytics-SelfService.md

```diff
@@ -0,0 +1,117 @@
+
+Also, many organizations have organized their data platform architecture and teams around functional responsibilities and pipeline stages that need to be applied to the various data assets of an organization. As a result, specialized teams within a data platform own an orthogonal solution such as ingestion, cleansing, aggregation or serving. This organizational and architectural concept leads to a dramatic loss of velocity, because when data consumers on the serving layer require new data assets to be onboarded or functional changes to be implemented for a specific data asset the following processes will need to be followed o safely roll out these changes:
+
+- The Data Consumer must submit a ticket to the functional teams being responsible for the respective pipeline stages.
+- Synchronization is now required between the functional teams as new ingestion services will be required, which will lead to changes required in the data cleansing layer, which will lead to changes on the data aggregation layer, which will again cause changes to be implemented on the serving layer. In summary, all pipeline stages may be impacted by the requested changes and clear impact on the processing staged will not be visible to any of these teams as no one overviews the end-to-end lifecycle.
+- In addition to the synchronization that is required between the various teams, the teams must design a very well-defined release plan in order to not impact existing consumers or pipelines. This dependency management further increases the management overhead.
+- All teams included in the requested change are most likely no subject matter experts for the specific data asset. Hence, additional consultation may be required to understand new dataset features or parameter values.
+- After applying all changes, the Data Consumer needs to be notified that the data asset is ready to be consumed.
+
+If we now take into consideration that within a large organization there is not a single data consumer but thousands of data consumers, the process described above makes clear why velocity is heavily impacted by such an architectural and organizational pattern. The centralized teams quickly become a bottleneck for the business units, which will ultimately result in slower innovations on the platform, limited effectiveness of data consumers and possibly even to a situation where individual business units decide to build their own data platform.
+
+## Methods of scaling in Enterprise-Scale Analytics
+
+![Enterprise-Scale Analytics - Multiple Data Landing Zone Pattern](/docs/images/MultipleLandingZones.png)
+
+Enterprise-Scale Analytics (ESA) uses two core concepts to overcome the issues mentioned in the [introduction](#introduction) above:
```

Comment:
Please consider rephrasing the sentence to "Enterprise-Scale Analytics (ESA) uses two core concepts to address scaling challenges:"